targets/TARGET_STM/TARGET_STM32L1/TARGET_IM880B/PeripheralPins.c

```diff
@@ -0,0 +1,129 @@
+/* mbed Microcontroller Library
+ *******************************************************************************
+ * Copyright (c) 2016, STMicroelectronics
+ * All rights reserved.
```

Comment:
I understand these files were copied from elsewhere or other target. Can you add SPDX identifier for new files (we should still fix  the old files). If you look at SPI.h or other files in platform folder, they contain

`SPDX-License-Identifier: Apache-2.0` , in this case it would be BSD I believe. please add to the new files

Reply:
Thanks for your feedback.

Is the "SPDX-License-Identifier: BSD-3-Clause" identifier correct?
Is this only for the "PeripheralPins.c" file required?

Reply:
Any new file should contain it (it's stated in our license docs guide).

Reply:
Yes, BSD-3-Clause should be the one for this file

Reply:
SPDX-License-Identifier added